src/app/modules/imap-server/imap-server.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { ImapMailsService } from '../../service/imapemails.service';
+
+@Component({
+    selector: 'app-imap-server',
+    templateUrl: './imap-server.component.html',
+    styleUrls: ['./imap-server.component.scss']
+})
+export class ImapServerComponent implements OnInit {
+    imapsettingjson: any[];
+    loading: boolean;
+    constructor(public imapservices: ImapMailsService) {}
+
+    ngOnInit() {
+        this.imapsettingjson = [{
+                '_id': '2398dskj3298',
```

Comment:
i have added imap_setting_json to check demo list. once i will get data through api i will assign imapsettingjson to that response